lib/api/v3/utilities/custom_field_injector.rb

```diff
@@ -275,6 +275,11 @@ def inject_property_value(custom_field, config)
                           setter: property_value_setter_for(custom_field),
                           cache_if: config[:cache_if],
                           render_nil: true
+
+          @class.property :"#{custom_field.attribute_name}errors",
```

Comment:
I understood the capitalisation problem - `custom_field.attribute_name` is in underscored form `custom_field_123`, so with `errors` it will be `custom_field_123errors` which is apparently converted to `customField123errors`, but with underscore it gets correct capitalizstion:

```suggestion
          @class.property :"#{custom_field.attribute_name}_errors",
```

Reply:
So that's how it is. Sure then, let's do it 👍